VERSION

```diff
@@ -1 +1 @@
-0.74-2
+krcroft/cdda/master
```

Comment:
I don't know what is the purpose of this VERSION file (and value 0.74-2 indicates, that upstream does not know either - trunk branch should have value 0.75-1 here or something). Regardless, definitely it should not be changed to name of your branch name :)

Reply:
Will remove this. I was also confused if/where it's used, but I didn't want to act like I was 'releasing my own 0.74' so felt compelled to change it.